From 0c15938e9628e524aa8ba37561d20789d41ef9e9 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Tue, 28 Mar 2023 09:15:25 +0530
Subject: [PATCH] Enabling kernel configs related to Android U

This patch will enable the required configs for Android U

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 .../config-lts/lts2021-chromium/x86_64_defconfig       | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig b/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig
index eee34ac..814b401 100644
--- a/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig
+++ b/groups/kernel/gmin64/config-lts/lts2021-chromium/x86_64_defconfig
@@ -6683,7 +6683,15 @@ CONFIG_LSM="selinux"
 # Memory initialization
 #
 CONFIG_CC_HAS_AUTO_VAR_INIT_PATTERN=y
-CONFIG_CC_HAS_AUTO_VAR_INIT_ZERO_ENABLER=y
+CONFIG_CC_HAS_AUTO_VAR_INIT_ZERO_BARE=y
+CONFIG_CC_HAS_RETURN_THUNK=y
+CONFIG_CC_HAS_SLS=y
+CONFIG_CC_HAS_ZERO_CALL_USED_REGS=y
+CONFIG_CPU_UNRET_ENTRY=y
+CONFIG_RETHUNK=y
+CONFIG_SLS=n
+CONFIG_ZERO_CALL_USED_REGS=n
+
 CONFIG_CC_HAS_AUTO_VAR_INIT_ZERO=y
 # CONFIG_INIT_STACK_NONE is not set
 # CONFIG_INIT_STACK_ALL_PATTERN is not set
-- 
2.17.1

